ucte/ucte-converter/pom.xml

```diff
@@ -53,6 +53,11 @@
             <artifactId>powsybl-iidm-converter-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>powsybl-iidm-scripting</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
This is not the correct way to do it: the post-processor should be discovered at runtime using the ServiceLoader. You should have a look to the `Importers` class to see how it works. The idea is to have something similar for exporters.